packages/react/src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -215,6 +216,7 @@ const TableToolbar = ({
       // TODO: remove deprecated 'testID' in v3
       data-testid={testID || testId}
       className={classnames(`${iotPrefix}--table-toolbar`, className)}
+      aria-label={i18n.toolbarLabelAria || secondaryTitle ? `${secondaryTitle} Toolbar` : undefined}
```

Comment:
Hi @TheSharpieOne. Thanks for the PR! Since this could change aria-lables that are out in the wild I think we should just use undefined if `toolbarlabelAria` is not present.

Reply:
Since aria-label could _not_ be specified, the _only_ value in the wild would be `"data table toolbar"`, which is not a sane default. So it's not like the default truly is `undefined`. Default values should be useful so that users don't need to specify every prop every time (only needing to use props such as this for specials cases (e.g. i18n)).
Look at the changed snapshots and tell me this was not an improvement in a11y for the default case providing `secondaryTitle`.
I can mark this as a breaking change if you would like (carbon react doesn't seem to believe changes like this are considered breaking.)

Reply:
It is an improvement, and I'll let David answer your question, but it is also a known problem for some of our clients that changes to default labels and roles etc are not considered breaking since they are sometimes used in automated tests that fail after the change.

Reply:
_rant_
Yes, I believe it is a breaking change as well. It will definitely break snapshots (as seen by snapshots changing in the PR.) I just wasn't sure if this repo was more aligned with carbon react in it's definition of a breaking change since it's also within the carbon-design-system org.
We have had carbon react break us many times. Here are the most recent:
[changes that prevent onChange callbacks from be called](https://github.com/carbon-design-system/carbon/pull/8018) which [break validation](https://github.com/carbon-design-system/carbon/issues/9337).
[changes that add internal validation which affect component rendered output](https://github.com/carbon-design-system/carbon/pull/9132) that cause visual changes.
_/rant_

I updated the commit message to indicate it is a breaking change. I included mitigation steps for users to preserve the current functionality to avoid changes.